app/components/Utils/TimeSince.jsx

```diff
@@ -37,7 +37,8 @@ export class TimeSince extends React.PureComponent {
     const { time, locale, dispatch, addSuffix = true, isDateTime = true, ...props } = this.props
     const localeObj = locales[locale]
     const dateFormat = isDateTime ? localeObj.defaultDateTimeFormat : localeObj.defaultDateFormat
-    const timeAsDate = time && parseISO(time)
+    const timeAsDate = time
```

Comment:
Let's make this backward compatible, it seems we're using a string in the videos list

```suggestion
    const timeAsDate = typeof time === 'string' ? parseISO(time) : time
```

![image](https://user-images.githubusercontent.com/1556356/108737233-b1660580-7532-11eb-9f8a-5ec0a6880250.png)